python/marvin/utils/datamodel/query/base.py

```diff
@@ -878,11 +884,23 @@ def _in_form(self):
 # # Get the Common Parameters from the filelist
 
 def get_params():
-    bestpath = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../../data',
-                            'query_params_best.cfg')
+
+    bestpath = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../../data', 'query_params_best.cfg')
+
     if os.path.isfile(bestpath):
         with open(bestpath, 'r') as stream:
             bestparams = yaml.load(stream, Loader=yamlordereddictloader.Loader)
+
```

Comment:
I'd like to change this so we're not managing multiple redundant files.  I've changed this to just remove spaxel properties from the `bestparams` dictionary.

Reply:
Sure, that was a hack because I didn't know how to to it better.